networkx/algorithms/operators/tests/test_all.py

```diff
@@ -47,6 +47,7 @@ def test_intersection_all():
     I = nx.intersection_all([G, H, R])
     assert set(I.nodes()) == {1, 2, 3, 4}
     assert sorted(I.edges()) == [(2, 3)]
+    assert I.graph["awesome"]
```

Comment:
I thought it'd be nice if `intersection_all` also retains attrs (like the other functions in this module do), but maybe that's not completely correct 🤔

Reply:
Would also be cool if `intersect_all` would retain node/edge data. For example by pulling nodes and edges from the last graph with `data=True`. Probably warrants a separate PR though.

Reply:
Node/edge data is always fussy.  If the user wants the intersection of two edges with the same endpoints but differing data, should the intersection be empty?  I suppose that it would really depend on individual cases, so I agree, much better to have that conversation on a separate PR.